lib/stacks/eks-blueprint-stack.ts

```diff
@@ -54,9 +53,22 @@ export class EksBlueprintProps {
      * Control Plane log types to be enabled (if not passed, none)
      * If wrong types are included, will throw an error.
      */
-    readonly enableControlPlaneLogTypes?: string[];
+    readonly enableControlPlaneLogTypes?: ControlPlaneLogTypes[];
 }
 
+const blueprintPropsContraints: ConstraintsType<EksBlueprintProps> = {
+    
+    id: new StringConstraint(1,63),
+    name: new StringConstraint(1,63)
+};
+
+export enum ControlPlaneLogTypes {
```

Comment:
minor: Name of the type should be singular. `ControlPlaneLogType`

Reply:
Is it singular since its called one at a time like ControlPlaneLogType.api and not a list of things after the dot? Even though it holds an array of various types and many can be called, but only one at a time?